backend/src/SM.Service/Extensions/EventStoreExtensions.cs

```diff
@@ -42,5 +44,28 @@ public static IMessage ReadMessage(this RecordedEvent recordedEvent)
 
             return null;
         }
+
+        public static async Task<PatternItems> GetUserPatterns(this string userId)
+        {
+            var (patternsByOwnerProjection, _) = await Cluster.GetAsync(ActorKind.PatternsByOwnerProjection, ActorKind.PatternsByOwnerProjection);
+
+            var query = new GetPatternItems {RequestId = Guid.NewGuid().ToString(), OwnerId = userId, Skip = 0, Take = 100};
+
+            while (true)
+            {
+//                if (some timeout condition) throw new TimeoutException();
```

Comment:
Should probably implement some real condition. Maybe wait for 30 seconds or for a minute? And error message with some descriptive message as I suggested earlier.